src/commands/rules/enableRules.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { ApiCenterService } from "../../azure/ApiCenter/ApiCenterService";
+import { ApiCenterRulesetConfig, ApiCenterRulesetImport } from "../../azure/ApiCenter/contracts";
+import { ext } from "../../extensionVariables";
+import { RulesTreeItem } from "../../tree/rules/RulesTreeItem";
+import { UiStrings } from "../../uiStrings";
+import { zipFolderToBuffer } from "../../utils/zipUtils";
+
+export async function enableRules(context: IActionContext, node: RulesTreeItem) {
+    const resourceGroupName = getResourceGroupFromId(node.apiCenter.id);
+    const apiCenterService = new ApiCenterService(node.parent?.subscription!, resourceGroupName, node.apiCenter.name);
+
+    const apiCenterRulesetConfig: ApiCenterRulesetConfig = {
+        properties: {
+            analyzerVersion: "1.0.0",
```

Comment:
these are all hard-coded?

Reply:
Yes, and currently the values are not awared and contriled by end users.